app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<div>
+  <h1>TO-DO LIST</h1>
+  <ul>
+    <% count = 1 %>
+    <% @tasks.each do |task| %>
```

Comment:
Instead of declaring `count` above, you might want to look into Ruby's `each_with_index`